dialdbg/README.md

```diff
@@ -0,0 +1,47 @@
+# Viam Dial Debugger
+A CLI tool to give information on how Viam rust utils' dial function makes connections.
+
+## Installation
+TODO(RSDK-3446): ensure installation constructions are correct.
+
+viam-dialdbg can be installed via homebrew.
+
+``` shell
+brew install viam-dialdbg
```

Comment:
Is this true?? if so, exciting! However I'm getting a `No formulae or casks found for viam-dialdbg` error when running that.

Reply:
It's not true 😢 ; see the TODO above. This is my predicted shell invocation for eventually installing `viam-dialdbg` via `brew`. Will handle in https://viam.atlassian.net/browse/RSDK-3446.